x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -31,6 +32,22 @@ const technicalPreviewLabel = i18n.translate(
  * uiSettings definitions for Observability.
  */
 export const uiSettings: Record<string, UiSettingsParams<boolean | number | string>> = {
+  [enableNewSyntheticsView]: {
+    category: [observabilityFeatureId],
+    name: i18n.translate('xpack.observability.enableNewSyntheticsViewExperimentName', {
+      defaultMessage: 'Enable new synthetic monitoring view',
+    }),
+    value: false,
+    description: i18n.translate(
+      'xpack.observability.enableNewSyntheticsViewExperimentDescription',
+      {
+        defaultMessage:
+          'Enable new synthetic monitoring view in observability. A page refresh is required for the setting to be applied.',
```

Comment:
You could simplify the second sentence. What do you think about an instructive tone? 

```suggestion
          'Enable new synthetic monitoring view in observability. Refresh the page to apply the setting.',
```

Reply:
Technically you don't even need to mention that a page refresh is required since the advanced settings UI will already notify you when you go to save changes. But I realize that there are some other settings in Kibana where we are explicitly saying this in the description, so it probably doesn't matter either way 🤷